query-engine/connector-test-kit-rs/query-engine-tests/tests/new/regressions/prisma_21369.rs

```diff
@@ -0,0 +1,17 @@
+use query_engine_tests::*;
+
+#[test_suite(schema(generic), exclude(MongoDb))]
+mod prisma_21369 {
+    #[connector_test]
+    async fn select_null_works(runner: Runner) -> TestResult<()> {
+        let query = fmt_query_raw("SELECT null as result", []);
```

Comment:
We don't run PlanetScale tests in CI in prisma-engines yet, but it's still useful to have this test for all connectors anyway, driver adapters or not.

The fix is manually tested by me and by the original reporter, who confirmed it is working in https://github.com/prisma/prisma/issues/21369#issuecomment-1749279407. I will add a regression test on the client side as well since we run PlanetScale adapter tests there.